share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/AfuncW.py

```diff
@@ -0,0 +1,112 @@
+"""

```

Comment:
Please add the license header similar to (https://github.com/ComputationalRadiationPhysics/picongpu/blob/59e9b53605f9a5c1bf271eeb055bc74370a99052/lib/python/picongpu/plugins/data/phase_space.py)

Reply:
done